TF_Settings_Web/src/Components/Pages/Page.tsx

```diff
@@ -1,22 +1,15 @@
-import { Component } from "react";
-
-interface PageProps {
-
-}
-interface PageState {
-
-}
-
-export abstract class Page<TProps extends PageProps = PageProps,
-                  TState extends PageState = PageState>
-                    extends Component<TProps, TState> {
-
-    render () {
-        return (
-            <div>
-                This is the base Page Render.
-                You should not be seeing this.
-            </div>
-        );
+import { Component } from 'react';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
is there a way to disable things like this per-file to avoid repeated comment lines of eslint?

Reply:
Yes there is - addind `/* eslint-disable @typescript-eslint/no-empty-interface */` at the top of the file will do that